Gopkg.toml

```diff
@@ -68,3 +68,19 @@
 [[constraint]]
   branch = "master"
   name = "github.com/lxn/walk"
+
+[[constraint]]
+  name = "github.com/jaypipes/ghw"
+  version = "0.3.0"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/vcraescu/go-xrandr"
+
+[[constraint]]
+  name = "github.com/gentlemanautomaton/windevice"
+  revision = "de21ffdab1a3b60205fe0782a9df61feb34367f1"
+
+[[constraint]]
+  branch = "master"
```

Comment:
Shouldn't we ping to a version or commit instead simply `master`?

Reply:
done